.backportrc.json

```
@@ -1,6 +1,6 @@
 {
   "upstream": "elastic/beats",
-  "branches": [ { "name": "7.x", "checked": true }, "7.11", "7.10" ],
+  "branches": [ { "name": "7.x", "checked": true }, "7.11", "7.11", "7.10" ],
```

We already have 7.11 in the list. I guess we need to remove 7.10?

---

yes, we can remove 7.10

```suggestion
  "branches": [ { "name": "7.x", "checked": true }, "7.11" ],
```